### ISSUE TITLE
[Accessibility]: Sourcegraph setup is requiring dev-private repository but it is not present

### ISSUE THREAD

**Issue by ibilalkayy** [sourcegraph/sourcegraph#58818]
### Problem description

While setting up the sourcegraph through the `sg setup` command, I noticed that it requires a **dev-private** repository with a link ``github.com/sourcegraph/dev-private`` but when I try to open this link, it says that the repository is not present.

### Expected behavior

There could be two scenarios.

1. Either the **dev-private** repository is completely deleted or replaced with another repository whose name is not mentioned in the `sg setup` command.
2. Or it is not required anymore.

In both cases, a PR is required to either remove its link or update it with a new link.

### sg setup screenshot

![Screenshot from 2023-12-07 11-23-05](https://github.com/sourcegraph/sourcegraph/assets/64713734/bb7c9093-bee0-4200-be78-5c697e8f7859)

### The repository is not present

![Screenshot from 2023-12-07 11-21-17](https://github.com/sourcegraph/sourcegraph/assets/64713734/785a6720-a43d-4755-bbe0-9f1440bc3069)

**Comment by icbd** [comment]
I also encountered the same problem. 

Maybe the configuration documentation should be updated for community developers.

**Comment by leijun507** [comment]
Do you have a solution

**Comment by ibilalkayy** [comment]
I don't think I have a solution. If these things are not for the community developers, then why they are public.

**Comment by jhchabran** [comment]
Hey folks, was brought here from another issue, sharing my response: https://github.com/sourcegraph/sourcegraph/issues/61749#issuecomment-2095413593 

I'm closing the issue, as this is a "won't fix".
